### PR TITLE
Added a low-level API section to tf.data lab + a minor fix

### PR DESCRIPTION
**Pull request by takumiohym**
- Added a low-level tf.data section to reflect https://github.com/GoogleCloudPlatform/asl-ml-immersion/pull/195
- Minor fix: modified the shuffling section to reflect the default behavior of the `make_csv_dataset` function

### CODE REVIEW

**Review by sanjanalreddy — APPROVED** [review]
This is a clear and simple change. Thanks Takumi! Minor suggestions.

File: notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb [hunk 1]
```diff
@@ -248,12 +248,17 @@
    "cell_type": "markdown",
```
[the remark this answers is not in the document]
Comment (BenoitDherin) [reply notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb:248]: <p>+1</p><p> </p><p>Nit:</p><p> </p><ul><li> a additional argument -&gt; an additional argument (?)</li></ul>

[the remark this answers is not in the document]
Comment (BenoitDherin) [reply notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb:248]: <p>+1 for instance, a good example would be maybe whey we can not use the <code>tf.data.experimental.make_csv_dataset</code> function in the sequential lab.  </p>

Comment (BenoitDherin) [inline notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb:248]: <ul><li>"This function includes many convenient features, so the data is easy to work with. This includes:"</li></ul><p>--&gt; "This function has many convenient features:"</p><p> </p><p> </p><ul><li>This function returns --&gt; It returns</li></ul><p> </p><ul><li>What do mean by "Using the column headers as dictionary keys" ? Rephrase maybe.</li></ul><p> </p><ul><li>"Automatically determining the type of each column." --&gt; "It automatically determines the type of each column."</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (BenoitDherin) [inline notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb:248]: <p>Good catch!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

**Review by BenoitDherin — APPROVED** [review]
LGTM

**Pull request merged**